db/cassandra.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gocql/gocql"
+)
+
+// Employee struct
+type User struct {
```

Comment:
All comments of public structures should start with their name, e.g. `User is a ...`

Reply:
done